lib/matplotlib/dates.py

```diff
@@ -793,7 +797,7 @@ def __setstate__(self, state):
 
 class DateLocator(ticker.Locator):
     """
-    Determines the tick locations when plotting dates.
+    Determines the tick locations when plotting dates. This class is subclassed by other Locators and is not meant to be used on its own.
```

Comment:
Put the second line by itself.  (See first sentence of https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).